mlir-sycl/include/mlir/Dialect/SYCL/IR/SYCLOps.td

```diff
@@ -269,9 +270,9 @@ def SYCL_MinimumType : SYCL_Type<"Minimum", "minimum"> {
 def SYCL_MultiPtrType : SYCL_Type<"MultiPtr", "multi_ptr"> {
   let parameters = (ins "::mlir::Type":$dataType,
                         "mlir::sycl::AccessAddrSpace":$addrSpace,
-                        "mlir::sycl::DecoratedAccess":$decAccess,
+                        "mlir::sycl::AccessDecorated":$decAccess,
```

Comment:
[nit]: sometime we use "mlir::" and sometime "::mlir::". I think, we should use "::mlir::" for consistency and so that we ensure that the `mlir` namespace is the top level namespace.